x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/dataset_quality.tsx

```diff
@@ -41,13 +42,15 @@ export const createDatasetQuality = ({
     );
 
     return (
-      <DatasetQualityContext.Provider value={datasetQualityProviderValue}>
-        <SummaryPanelProvider>
-          <KibanaContextProviderForPlugin>
-            <DatasetQuality />
-          </KibanaContextProviderForPlugin>
-        </SummaryPanelProvider>
-      </DatasetQualityContext.Provider>
+      <PerformanceContextProvider>
```

Comment:
Is `PerformanceContextProvider` needed in both here and `application.tsx`?

Reply:
yup, this is what I'm figuring out with Kate.
Dataset quality plugin is not aware that a provider have been placed outside its context